core/include/detray/propagator/propagator.hpp

```diff
@@ -56,15 +45,20 @@ struct propagator {
         : _stepper(std::move(s)), _navigator(std::move(n)) {}
 
     /** Propagation that state aggregates a stepping and a navigation state */
+    template <typename actor_states_t = bool>
```

Comment:
Can we remove this `actor_state_t` from template parameter and use `actor_chaint_t::state` instead?

Reply:
Good idea!